product.js

```diff
@@ -0,0 +1,40 @@
+const arr = [3, 7, -13, -4];
+
+function productMultiply(arr) {
+    let negativeCount = 0;
+
+    if(arr.includes(0)) return 'The product is 0';
+
+    for (let i = 0; i < arr.length; i++) {
+        if (arr[i] < 0) {
+            negativeCount++;
+        }
+    }
+
+    if (negativeCount % 2 === 0) {
```

Comment:
0 is already a falsy value
```suggestion
    if (!(negativeCount % 2)) {
```

Reply:
done)